config.py

```diff
@@ -45,8 +45,8 @@ def generate_unique_email(email, uuid):
         "govuk_alerts": "http://localhost:6017/alerts",
     },
     "development": {
-        "api": "https://api.dev.emergency-alerts.service.gov.uk/",
-        "admin": "https://admin.dev.emergency-alerts.service.gov.uk/",
+        "api": "https://api.dev.emergency-alerts.service.gov.uk",
```

Comment:
Not something that I'd say needs amending in this PR specifically, as it's a separate issue - we'll need to have a think on how this is configured to target a particular developer's environment (we could probably pass in a variable in the CLI call made by the application pipelines to set that)

Reply:
Ticket created to address the above issue: https://gds-ea.atlassian.net/browse/EAS-1984